packages/firestore/src/api/aggregate.ts

```diff
@@ -17,23 +17,43 @@
 
 import { Query } from '../api';
 import { firestoreClientRunCountQuery } from '../core/firestore_client';
-import { AggregateField, AggregateQuerySnapshot } from '../lite-api/aggregate';
+import {
+  AggregateField,
+  AggregateQuerySnapshot
+} from '../lite-api/aggregate_types';
 import { cast } from '../util/input_validation';
 
 import { ensureFirestoreConfigured, Firestore } from './database';
+import { ExpUserDataWriter } from './reference_impl';
+
+export { aggregateQuerySnapshotEqual } from '../lite-api/aggregate';
 
 /**
- * Executes the query and returns the results as a `AggregateQuerySnapshot` from the
- * server. Returns an error if the network is not available.
+ * Calculates the number of documents in the result set of the given query,
```

Comment:
I think all of this comment is copied in FS lite, so all feedback applies there too, thanks!

Reply:
PR created: https://github.com/firebase/firebase-js-sdk/pull/7933